models/ed/tests/Rcheck_reference.log

```diff
@@ -205,17 +69,101 @@ Extensions’ manual.
 * checking for unstated dependencies in examples ... OK
 * checking contents of ‘data’ directory ... OK
 * checking data for non-ASCII characters ... OK
+* checking LazyData ... OK
 * checking data for ASCII and uncompressed saves ... OK
+* checking R/sysdata.rda ... OK
 * checking files in ‘vignettes’ ... WARNING
 Files in the 'vignettes' directory but no files in 'inst/doc':
-  ‘running_ed_from_R.Rmd’
+  ‘ed_run_data/EDI.zip’,
```

Comment:
Are these files that need to be distributed with the package, or are they leftovers from vignette building that ought to be deleted?

Reply:
It looks like that very large folder of .h5 files is used in the vignette. I put it in .Rbuildignore for now, but if there isn't already an issue open about fixing up this vignette, you should open one.  There's no way CRAN will like all those .h5 files being necessary for a vignette and there are other warnings from `check()` about it.

Reply:
Good call. I dropped the vignette cleanup into a fresh issue: https://github.com/PecanProject/pecan/issues/3141